### ISSUE TITLE
Would it be possible to reinstate the wiki and fix the links?

### ISSUE THREAD

**Issue by mrexodia** [cpp-pm/hunter#363]
While trying to use Hunter for the first time recently I ran into a bunch of dead links (both in the CMake logs and in the documentation). Most are to `github.com/ruslo` and point to archived repositories.

I found that `git clone https://github.com/ruslo/hunter.wiki` still works and I have a copy of the wiki, but I don't have access to push to the original contents there. I found `docs/old-wiki`, but it doesn't appear to be linked correctly.

A list of dead URLs I found so far:

- `hunter.sh`
- `github.com/ruslo/hunter/wiki`

**Comment by rbsheth** [comment]
Yes we should probably do that. But I would like to move the Wiki entries to the readthedocs site to consolidate.
